### DISCUSSION TITLE
[Arcropolis 3.14.0] Steve vs King K Rool crash

### DISCUSSION THREAD

**Discussion by RyufromStreets** [Raytwo/ARCropolis#544] [Issues]
**System Firmware Version:**
16.1.0

**Atmosphere Version:**
1.5.5

**Smash Version:**
13.0.1

**ARCropolis Version:**
3.14.0

**Can the crash be consistently reproduced?**
Yes

**At what moment does the crash happen?**
Whilst fighting steve online (it could be either quick play or an arena match) and steve uses anvil (down aerial) and King K Rool uses up smash or up aerial

**Details:**
I first experienced this crash while fighting a random quick play steve. After the game crashed i called on friend to join my online lobby so that we could keep experimenting to see if we could reproduce the crash scenario which we did. We tried it several times, all the while I was removing mods 1 by 1 to see if one of them was the culprit. After removing the sephiroth based steve mod and the Battlefield stage mod, me and my friend could no longer reproduce the event so I considered the case closed and moved on. The next day the crash would occur again while i was playing quickplay.

**Plugins turned on:**
libarcropolis.nro
libone_slot_eff.nro
libarena_latency_slider.nro
libsmashline_hook
libkamui_effect
libnro_hook
libcss_preserve.nro
libparam_hook
libnn_hid_hook.nro
libtraining_modpack
(i know there is a lot here but I'm pretty sure the only plugin that the mods on display depend upon is arcropolis)

**Crash report:**
[01693808738_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/12521441/01693808738_01006a800016e000.log)

[01693809425_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/12521450/01693809425_01006a800016e000.log)

[01693812435_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/12521452/01693812435_01006a800016e000.log)

[01693904584_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/12521533/01693904584_01006a800016e000.log)

Also here is a acropolis log of the most recent crash I don't if it will be helpful
[2023-09-05_05-50-45.log](https://github.com/Raytwo/ARCropolis/files/12521897/2023-09-05_05-50-45.log)

Reproducible Modpack:
[Mods on display](https://www.mediafire.com/file/uh2v225pvlbrhci/mods_on_display.7z/file)

**Moment of Crash:**

![01693808738_01006a800016e000](https://github.com/Raytwo/ARCropolis/assets/104103776/55dfae1a-cbed-4752-8949-a75d7b074da1)
![01693809425_01006a800016e000](https://github.com/Raytwo/ARCropolis/assets/104103776/8f998232-955b-4ce9-ba15-382878823f14)
![01693812435_01006a800016e000](https://github.com/Raytwo/ARCropolis/assets/104103776/ca5e4d75-c0da-4f74-b9f7-ec649903d308)
![01693904584_01006a800016e000](https://github.com/Raytwo/ARCropolis/assets/104103776/e132c426-38d9-4159-b677-41c4c9b073e8)

**Comment by jozz024** [comment]
turn off every other plugin and see if you still get the issue

**Comment by RyufromStreets** [comment]
So in my ignorance, I assumed that because the the mods on display didn't depend on the plugins to work then the the plugins themselves weren't affecting the game in anyway. I don't know why I thought this. I did disable the plugins (and most mods as well) and I believe the issue was being caused by libnro_hook.nro, libparam_hook.nro, or libtraining_modpack.nro. I did not experience any crashing while playing against steve since I disabled those plugins.
